src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+  let result = 0;
+
+  if (days >= 1 && days < 3) {
+    result = total;
+  } else if (days >= 3 && days < 7) {
+    result = total - 20;
```

Comment:
Move 20 and 50 to discount variables.

Reply:
Better to move 7 days if above the 3 days if. 
If you do this you can replace `(days >= 3 && days < 7) ` to `(days >= 3)`